source/operations/cert-manager/cert-manager-operator.rst

```diff
@@ -0,0 +1,239 @@
+Create this CA certificate using cert-manager.
+
+.. important::
+
+   This CA certificate should exist *before* installing MinIO Operator.
+
+1. If it does not exist, create the ``minio-operator`` namespace
+
+   .. code-block:: shell
+      :class: copyable
+
+      kubectl create ns minio-operator
+
+2. Request a new Certificate with ``spec.isCA: true`` specified. 
+
+   This certificate serves as the :abbr:`CA (Certificate Authority)` for the `minio-operator` namespace.
+
+   Create a file called ``operator-ca-tls-secret.yaml`` with the following contents:
+
+   .. code-block:: yaml
+      :class: copyable
+      :emphasize-lines: 7,8
+
+      # operator-ca-tls-secret.yaml
+      apiVersion: cert-manager.io/v1
+      kind: Certificate
+      metadata:
+        name: minio-operator-ca-certificate
+        namespace: minio-operator
+      spec:
+        isCA: true
+        commonName: operator
+        secretName: operator-ca-tls
+        duration: 70128h # 8y
+        privateKey:
+          algorithm: ECDSA
+          size: 256
+        issuerRef:
+          name: selfsigned-root
+          kind: ClusterIssuer
+          group: cert-manager.io
+
+3. Apply the resource to the cluster
+   
+   .. code-block:: shell
+      :class: copyable
+
+      kubectl apply -f operator-ca-tls-secret.yaml
+
+Kubernetes creates a new secret with the name ``operator-ca-tls`` in the ``minio-operator`` namespace.
+
+.. important::
+
+   Make sure to trust this certificate in any applications that need to interact with the MinIO Operator.
+
+
+2) Use the secret to create the ``Issuer``
+------------------------------------------
+
+Use the secret created above to add an ``Issuer`` resource for the ``minio-operator`` namespace.
+
+1. Create a file called ``operator-ca-issuer.yaml`` with the following contents: 
+
+   .. code-block:: yaml
+
+      # operator-ca-issuer.yaml
+      apiVersion: cert-manager.io/v1
+      kind: Issuer
+      metadata:
+        name: minio-operator-ca-issuer
+        namespace: minio-operator
+      spec:
+        ca:
+          secretName: operator-ca-tls
+
+
+2. Apply the resource to the cluster
+   
+   .. code-block:: shell
+
+      kubectl apply -f operator-ca-issuer.yaml
+
+3) Create TLS certificate
+-------------------------
+
+Now that the ``Issuer`` exists in the ``minio-operator`` namespace, cert-manager can add a certificate.
+
+The certificate from cert-manager must be valid for the following DNS domains:
+
+- ``sts``
+- ``sts.minio-operator.svc.``
+- ``sts.minio-operator.svc.<cluster domain>``
+
+  .. important::
+
+      Replace ``<cluster domain>`` with the actual value for your MinIO tenant.
+      ``cluster domain`` is the internal root DNS domain assigned in your Kubernetes cluster. 
+      Typically, this is ``cluster.local``, but confirm the value by checking your coredns configuration for the correct value for your Kubernetes cluster. 
```

Comment:
Is coredns here referring to a process or the CoreDNS product?

Reply:
The product, I think.